mocks/apiHandlers/todoHandlers.ts

```
@@ -0,0 +1,117 @@
+      (todoRes) => todoRes.todoResultId !== targetId,
+    );
+
+    // todo에서 처리
+    myMainDB.todos = myMainDB.todos.reduce(
+      (acc, todo: (typeof myMainDB.todos)[0]) => {
+        if (!todo.todoResultId?.includes(targetId)) return [...acc, todo];
+        return [
+          ...acc,
+          {
+            ...todo,
+            todoResultId: undefined,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)